tests/index.js

```diff
@@ -24,11 +24,30 @@ describe('all rule files should be exported by the plugin', () => {
 
 describe('rule documentation files have the correct content', () => {
   const MESSAGES = {
+    configs: '💼 This rule is enabled in the following [configs](https://github.com/jsx-eslint/eslint-plugin-react#shareable-configurations):',
+    configsOff: 'This rule is disabled in the following configs:',
     deprecated: '❌ This rule is deprecated.',
     fixable: '🔧 This rule is automatically fixable using the `--fix` [flag](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix) on the command line.',
     hasSuggestions: '💡 This rule provides editor [suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).',
   };
 
+  function getConfigsForRule(ruleName, checkForEnabled = true) {
```

Comment:
these tests run in node versions without default arguments, so we'll need this:
```suggestion
  function getConfigsForRule(ruleName, checkForEnabled) {
```

Reply:
Yeah, thanks for removing the default arg.